src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,32 @@
+import logo from "../../assets/logo.png";
+import { NavLink } from "react-router-dom";
+import { navigationIcons } from "./NavIconData";
+
+const NavBar = () => {
+  return (
+    <div className=" icons-center bl-black-200 flex h-full w-20 flex-col border pt-10">
+      <img src={logo} alt="logo" className="h-20 w-auto p-5" />
+
+      <div className="mt-16 p-2">
+        {navigationIcons.map((icon) => (
+          <NavLink key={icon.name} to={icon.href}>
+            {({ isActive }) => (
```

Comment:
Where is `isActive` defined? Can you add a test for `isActive`?

Reply:
isActive is a built-in variable in NavLink!  Will do!